coderd/workspaceagents.go

```diff
@@ -733,7 +735,8 @@ func (api *API) workspaceAgentConnection(rw http.ResponseWriter, r *http.Request
 	ctx := r.Context()
 
 	httpapi.Write(ctx, rw, http.StatusOK, codersdk.WorkspaceAgentConnectionInfo{
-		DERPMap: api.DERPMap,
+		DERPMap:                api.DERPMap,
+		AllowDirectConnections: !api.DeploymentValues.DERP.Config.DisableDirect.Value(),
```

Comment:
Can we call this DisableDirectConnections too for consistency?

Reply:
Agree here. I find it weird we go from `block direct (coderd flag) -> allow direct (coderd api) -> block direct (tailnet)`